workspace/utilities/lib/create-page-url.xsl

```diff
@@ -27,7 +27,7 @@
 					<xsl:with-param name="lg" select="$lg" />
 					<xsl:with-param name="url">
 						<xsl:choose>
-							<xsl:when test="$multi-langues = 'yes'">
+							<xsl:when test="$multilingual">
```

Comment:
I would prefer to see `$multilingual = true()` or `$is-multilingual` . Simply `$multilingual` seems un complete to me.

Reply:
+1

Reply:
$is-multilingual FTW 💃

Reply:
$is-multilingual it is then!